homework_5/Methods/GuessWordGame.java

```diff
@@ -1,5 +1,35 @@
 package Methods;
 
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+
 public class GuessWordGame {
-    
+    public static void main(String[] args){
+        Scanner scanner = new Scanner(System.in);
+        Random random = new Random();
+        String[] words = {"apple", "orange", "lemon", "banana", "apricot", "avocado" , "broccoli", "carrot", "cherry",
+                "garlic", "grape", "melon", "leak", "kiwi", "mango", "mushroom", "nut", "olive", " pea", "peanut",
+                "pear", "pepper", "pineapple", "pumpkin", "potato"};
+        String randomWord = words[random.nextInt(words.length)];
+        System.out.println(Arrays.toString(words));
+        System.out.print("Guess a word: ");
+        String guess_word = scanner.nextLine().toLowerCase();
+        while (!randomWord.equals(guess_word)){
+            StringBuilder stringBuilder = new StringBuilder ("###############");
+            int shortestWordLenght;
+            shortestWordLenght = randomWord.length() < guess_word.length() ?
```

Comment:
Could you please inline the declaration and initialization of a 'shortestWordLenght' variable?